kapacitor/ast.go

```diff
@@ -851,3 +851,51 @@ func extractExec(node *pipeline.AlertNode, rule *chronograf.AlertRule) {
 
 	rule.AlertNodes = append(rule.AlertNodes, alert)
 }
+
+func extractPushover(node *pipeline.AlertNode, rule *chronograf.AlertRule) {
+	if node.PushoverHandlers == nil {
```

Comment:
This will work for a `nil` slice, but not for an empty slice. You should check this like:

```go
if len(node.PushoverHandlers) == 0 {
    return
}
```

See [this playground](https://play.golang.org/p/FZDK-N7kJX) for a runnable example of the difference.